### PR TITLE
Added action to open update image wizard component using table actions

### PR DESCRIPTION
**Pull request by matoval**
Added action to open update image wizard component using table actions. If an image has a status of success show update action.

### CODE REVIEW

**Comment by boaz0** [comment]
@matoval today your team mate @resoluteCoder is going to review your PR.
If you need help with resolving conflicts - lemme know I will be happy to help you.
Again sorry for blocking you with this.

**Review by resoluteCoder — APPROVED** [review]
Looks good :+1:

**Comment by boaz0** [comment]
@matoval there is conflicts. can you rebase? 
@resoluteCoder after this PR will be conflict resolved, please review this again.

Thanks.

File: src/Routes/ImageManager/ImageTable.js [hunk 1]
```diff
@@ -208,24 +217,33 @@ const ImageTable = ({ filters, pagination, openWizard, clearFilters }) => {
...
-    }
-
-    return [
-      {
+    const actionsArray = [];
+    if (rowData?.isoURL !== undefined) {
+      actionsArray.push({
         title: (
-          <a
+          <SimpleListItem
```
Comment (boaz0) [inline src/Routes/ImageManager/ImageTable.js:224]: I think `Text` is better https://www.patternfly.org/v4/components/text

```js
<Text component="a" href={rowData.isoURL} ...>Download</Text> 
```

File: src/Routes/ImageManager/ImageTable.js [hunk 2]
```diff
@@ -33,7 +34,14 @@ import {
...
 import PropTypes from 'prop-types';
 
-const ImageTable = ({ filters, pagination, openWizard, clearFilters }) => {
+const ImageTable = ({
+  filters,
+  pagination,
+  clearFilters,
+  openCreateWizard,
+  openUpdateWizard,
+  setUpdateImageID,
```
Comment (boaz0) [inline src/Routes/ImageManager/ImageTable.js:43]: can we create openUpdateWizard that will both open the update wizard and set the update image ID?

**Review by boaz0 — CHANGES_REQUESTED** [review]
Why not changing

**Review by boaz0 — APPROVED** [review]

**Comment by boaz0** [comment]
@matoval small nitpick request - make your commit message less than 71 characters.
Thanks.

**Pull request merged**